### PR TITLE
arp-scan: add page

### PR DESCRIPTION
**Pull request by Neph0**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/arp-scan.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# arp-scan
+
+> Send ARP packets to hosts on the local network to scan which ones answer.
+
+- Scan the current local network:
+
+`arp-scan --localnet`
+
+- Scan an IP network with a custom bits mask:
```
Comment (sbrl) [inline pages/linux/arp-scan.md]: Shouldn't this be "bitmask"?

File: pages/linux/arp-scan.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+`arp-scan {{192.168.1.1}}/{{24}}`
+
+- Scan an IP network within a custom range:
+
+`arp-scan {{127.0.0.0}}-{{127.0.0.31}}`
+
+- Scan an IN network with a custom net mask:
```
Comment (sbrl) [inline pages/linux/arp-scan.md]: What's the difference between an `IP` network and an `IN` network?

File: pages/linux/arp-scan.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# arp-scan
+
+> Send ARP packets to hosts on the local network to scan which ones answer.
+
+- Scan the current local network:
```
Comment (sbrl) [inline pages/linux/arp-scan.md]: `IP` / `IN` are mentioned below, but not specified here.....?

File: pages/linux/arp-scan.md (hunk 2 above)
Comment (Neph0) [reply pages/linux/arp-scan.md]: The difference is a typo, my mistake

File: pages/linux/arp-scan.md (hunk 1 above)
Comment (Neph0) [reply pages/linux/arp-scan.md]: I think you're right, I'll fix this when I have some free time!

Comment (sbrl) [reply pages/linux/arp-scan.md]: Sure! Just @mention me in a comment once you've made the changes.

**Comment by sbrl** [comment]
Thanks, @Neph0!

**Comment by Neph0** [comment]
No problem, thanks to whoever started this great project :+1:

**Pull request merged**